pkg/ast/spl/spl.peg

```diff
@@ -556,7 +568,7 @@ FieldSelectBlock <- PIPE CMD_FIELDS op:("-" / "+")? EMPTY_OR_SPACE fields:FieldN
     return queryAggregator, nil
 }
 
-AggregatorBlock <- PIPE CMD_STATS aggs:AggregationList byFields:(GroupbyBlock)? options:(StatsOptions)?{
+CommonAggregatorBlock <- aggs:AggregationList byFields:(GroupbyBlock)? {
```

Comment:
I think it is better to keep this as `AggregatorBlock` and then rename the `stats` one as `StatsAggregatorBlock` or just `StatsBlock`.

Reply:
I think the point of this was to share some common code between splunk commands that do something like `<agg_function>(<field1>) by <field2>`, so I think what Vasu has is good